drizzlepac/pars/wfc3_header_hap.rules

```diff
@@ -358,10 +358,25 @@ EQUINOX   EQUINOX   first
 # Header Keyword Rules
 #
 ################################################################################
+#
+# Keywords which need to be deleted for MVM products
+# These are not relevant for exposures from multiple proposals
+#
+################################################################################
+<delete>  IPPPSSOO
+<delete>  ASN_ID
+<delete>  ASN_MTYP
+<delete>  ASN_TAB
+<delete>  GYROMODE
+<delete>  SUNANGLE
+<delete>  MOONANGLE
```

Comment:
Should this be MOONANGL?

Reply:
Fixed in this file as well as 'wfc3_mvm_header_hap.rules' in a625671.